apps/web/src/pages/integrations/components/ConnectIntegrationForm.tsx

```diff
@@ -1,16 +1,26 @@
 import { useEffect, useState, useReducer } from 'react';
 import { Controller, useForm } from 'react-hook-form';
 import styled from '@emotion/styled/macro';
-import { ChannelTypeEnum, ICredentialsDto, IConfigCredentials } from '@novu/shared';
-import { useMutation } from 'react-query';
+import { useMutation, useQuery } from 'react-query';
 import { showNotification } from '@mantine/notifications';
+import { useClipboard } from '@mantine/hooks';
 import { Image, useMantineColorScheme, Stack, Alert } from '@mantine/core';
-import { Button, colors, Switch, Text } from '../../../design-system';
+import { WarningOutlined } from '@ant-design/icons';
+import { ChannelTypeEnum, ICredentialsDto, IConfigCredentials } from '@novu/shared';
+import { Button, colors, Input, Switch, Text } from '../../../design-system';
 import { IIntegratedProvider } from '../IntegrationsStorePage';
-import { createIntegration, updateIntegration } from '../../../api/integration';
+import {
+  createIntegration,
+  getActiveIntegrations,
+  getWebhookSupportStatus,
+  updateIntegration,
+} from '../../../api/integration';
 import { Close } from '../../../design-system/icons/actions/Close';
 import { IntegrationInput } from './IntegrationInput';
-import { WarningOutlined } from '@ant-design/icons';
+import { API_ROOT } from '../../../config';
+import { useEnvController } from '../../../store/use-env-controller';
+import { useAuthController } from '../../../store/use-auth-controller';
```

Comment:
Maybe we could add an index exporter file to group the controller hooks of the web store.